Update to marketing handbook (#9448)
PostHog/posthog.com: contents/handbook/growth/marketing/index.md

showTitle: true
---

The marketing team has two core goals:

> 1. Increase awareness of PostHog, especially among people in our ideal customer profile.

Author: ```suggestion
1. Increase awareness of PostHog, especially among people in our ideal customer profile
```

2. Help developers and PostHog users flex their product muscles and be more successful

We do this by:

- Building a reputation for world-class content
- Having the coolest merch that people want to wear all the time
- Constantly working to improve our documentation
- Identifying where content can help users be more successful
- Proactively promoting the work of product teams
- Producing ad campaigns that people actually like and share 
- Holding a high bar for quality in everything we do
- Never being satisfied with how high that bar is
- Being weird, opinionated, and unafraid of being wrong
- Reacting quickly to opportunities whenever they arise
- Not being precious about our work and priorities
- Having a team of talented, technically literate writers (and Andy)
- Not relying on freelancers or guest contributors for content
- Avoiding tedious enterprise marketing nonsense (PDFs, gated content, webinars, etc.)

> - Just being a great vibe generally

Author: ```suggestion
```

showTitle: true
---

The marketing team has two core goals:

1. Increase awareness of PostHog, especially among people in our ideal customer profile.

> 2. Make engineers and PostHog users feel like badass ninjas by teaching them cool stuff.

Reviewer: > badass ninjas

A bit cringe in framing, IMHO. It's like posting a job ad asking for a rockstar.

Author: yeah, I figured the same tbh. I'll come up something else.

Reviewer 2: "flex their product muscle" (this is how I frame Product for Engineers)

We do this by:

- Building a reputation for world-class content
- Having the coolest merch that people want to wear all the time
- Constantly working to improve our documentation
- Identifying where content can help users be more successful
- Proactively promoting the work of product teams

> - Never telling them what to build

Reviewer 2: I don't know if this needs to be said. I feel like it is pretty clear and could turn off people giving feedback/making feature requests
```suggestion
```

- Producing ad campaigns that people actually like and share 
- Holding a high bar for quality in everything we do
- Never being satisfied with how high that bar is
- Being weird, opinionated, and unafraid of being wrong
- Reacting quickly to opportunities whenever they arise
- Not being precious about our work and priorities

> - Having a team of talented, technically literate writers (and Andy)

Reviewer 3: savage

> - Not overly relying on freelancers or guest contributors for content

Reviewer 2: I think we should aim to never "rely" on them (open to collaborating)
```suggestion
- Not relying on freelancers or guest contributors for content
```

- Avoiding tedious enterprise marketing nonsense (PDFs, gated content, webinars, etc.)

> - Just being a great vibe generally

Reviewer 3: This is a tiny bit 'how do you do fellow kids'

Author: I will remove if you feel strongly

Reviewer 3: Yeah, I think if you are a great vibe you don't declare it, you just are it

PostHog's [vision](/handbook/strategy/overview#long-term-vision-for-2026) is to IPO in 2026 with $100m ARR. This is what we think marketing & content at PostHog will look like at that point. 

### Things we want to be brilliant at
 
- **Word of mouth mindset:** By IPO, we want to have built a hugely successful company driven primarily by word of mouth, rather than paid ads or PR.

- **Speed:** We want to be highly reactive, low process, and reliant on other teams as little as possible to ship things. We want to get stuff wrong quickly, then iterate.

- **Helping PostHog users be successful:** Through our docs, tutorials, newsletter and beyond, we help PostHog users be more successful, both generally in their goals, but also more successful and powerful users of PostHog.

- **Content for our ideal customer:** We deliver genuinely useful insights about things those customers care about (can be purely product-related, but also general advice). We're pitching our content at ‘self-servers’.

> - **Documentation** We work with Website & Docs, and product teams, to maintain docs to a high standard, and ensure users can easily find answers to their questions.

Reviewer 2: ```suggestion
- **Documentation** We work with product teams to maintain update-to-date, high quality docs. We work with Website & Docs to ensure users can discover what they want to know. 
```
